src/System.Private.ServiceModel/tests/Scenarios/Bridge/.nuget/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="true" />
+  </solution>
+</configuration>
```

Comment:
NIT: Need newline. 
Question: do we really need to check in the Nuget.exe binary? Can we not piggyback off what's already in the repo to do this?